appsec/appsec.go

```diff
@@ -0,0 +1,25 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+// Package appsec provides application security features in the form of SDK
+// functions that can be manually called to monitor specific code paths and data
+package appsec
+
+import (
+	"golang.org/x/net/context"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation/httpsec"
+)
+
+// MonitorParsedHTTPBody performs rule matching on the *parsed* HTTP request body.
+// - body: the parsed HTTP body value to be used for rule matching.
+// - ctx: the context of the http.Request for which the matching is to be performed.
+// Note that some HTTP frameworks implement their own version of context.Context,
+// however this function expects this exact type to be used.
+// Also note that passing the HTTP request raw body instead of parsed will result
+// in inaccurate attack detection
+func MonitorParsedHTTPBody(ctx context.Context, body interface{}) {
+	httpsec.MonitorParsedBody(ctx, body)
```

Comment:
I doesn't compile, but as discussed privately, we should add:
```suggestion
	if !internal.Enabled() {
		// bonus: use sync.Once to log a debug message once, but I think it is fine not to.
		return
	}
	httpsec.MonitorParsedBody(ctx, body)
```
In order to better log in httpsec when the parent operation cannot be found, under the new assumption it should be called when appsec is disabled.